elixir/lib/mqtt_client_handler.ex

```diff
@@ -0,0 +1,37 @@
+defmodule MqttClientHandler do
+  @moduledoc """
+  MqttClientHandler
+  """
+  use Tortoise.Handler
+
+  def init(args) do
+    {:ok, args}
+  end
+
+  def connection(_status, state) do
+    {:ok, state}
+  end
+
+  @doc """
+  Get messages from topic /telemetry/transport/+
+  """
+  def handle_message(["telemetry", "transport", _track], payload, state) do
+    # decods json to a map
+    _decoded = Jason.decode!(payload)
+    {:ok, state}
```

Comment:
I ran the program according to the instructions, but there is no visible activity on the screen, so one is not sure if it actually works. I sniffed the traffic so I know it does :-) but it would be user friendlier if you added something like

```
    IO.puts(track)
    IO.inspect(decoded)
```
to the code, to show some positive confirmation that the program really receives data.

Reply:
Added Logger to see the messages